com.unity.render-pipelines.universal/Runtime/ShadowUtils.cs

```diff
@@ -119,10 +121,10 @@ public static bool ExtractPointLightMatrix(ref CullingResults cullResults, ref S
         }
 
         public static void RenderShadowSlice(CommandBuffer cmd, ref ScriptableRenderContext context,
-            ref ShadowSliceData shadowSliceData, ref ShadowDrawingSettings settings)
+            ref ShadowSliceData shadowSliceData, ref ShadowDrawingSettings settings, float depthBias)
```

Comment:
Public API breakage. One option is to add default parameters to avoid it. It would also be great to add the missing scripting API docs.